frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -12,10 +12,10 @@ import {
   useWorkflowStorageContext,
 } from "@clutch-sh/core";
 import styled from "@emotion/styled";
-import { Divider, LinearProgress } from "@material-ui/core";
-import AddIcon from "@material-ui/icons/Add";
-import InfoOutlinedIcon from "@material-ui/icons/InfoOutlined";
-import LayersOutlinedIcon from "@material-ui/icons/LayersOutlined";
+import AddIcon from "@mui/icons-material/Add";
```

Comment:
not seeing changes in this PR that might have caused this but seeing a bigger gap between the search bar and project list than there was prior. 

on `matv5` branch
![Screen Shot 2022-08-03 at 12 20 35 PM](https://user-images.githubusercontent.com/39421794/182660440-2fe2c741-3b03-446e-8ae9-aa98246ad7c4.png)

on `main` branch
![Screen Shot 2022-08-03 at 12 27 32 PM](https://user-images.githubusercontent.com/39421794/182660741-281c8d84-1605-4fda-a47f-244e5d0afd4b.png)

Reply:
This should now be fixed as part of the `matv5-core-input` branch